### PR TITLE
Feature create cronjobs

### PR DESCRIPTION
**Pull request by fccoelho**
Scripts "pega_tweets" e "capturachuva.py" are ready to be called  from CRON to solve the instability issues we are having with celerybeat.

after merging the server's crontab must be update to add an entry for the daily executionof these scripts.

### CODE REVIEW

File: crawlclima/capturachuva.py [hunk 1]
```diff
@@ -10,8 +13,14 @@
...
-pega_dados_cemaden.delay('RJ', week_ago, today, 'uf')
-pega_dados_cemaden.delay('MG', week_ago, today, 'uf')
+if today.isoweekday() != 5:
+    pega_dados_cemaden.delay('PR', week_ago, today, 'uf')
+    pega_dados_cemaden.delay('RJ', week_ago, today, 'uf')
+    pega_dados_cemaden.delay('MG', week_ago, today, 'uf')
+else:
+    pega_dados_cemaden.delay('PR', year_start, today, 'uf')
+    pega_dados_cemaden.delay('RJ', year_start, today, 'uf')
+    pega_dados_cemaden.delay('MG', year_start, today, 'uf')
```
Comment (israelst) [inline crawlclima/capturachuva.py]: Suggestion:
```python
date_from = week_ago if today.isoweekday() != 5 else year_start

pega_dados_cemaden.delay('PR', date_from, today, 'uf')
pega_dados_cemaden.delay('RJ', date_from, today, 'uf')
pega_dados_cemaden.delay('MG', date_from, today, 'uf')
```

File: crawlclima/pegatweets.py [hunk 2]
```diff
@@ -9,8 +11,17 @@
...
-today = datetime.fromordinal(date.today().toordinal())
-week_ago = datetime.fromordinal(date.today().toordinal())-timedelta(8)
+today = date.fromordinal(date.today().toordinal())
+week_ago = date.fromordinal(date.today().toordinal())-timedelta(8)
+year_start = date(date.today().year, 1, 1)
+
+with open("../municipios") as f:
+    municipios = f.read().split('\n')
+
+municipios = [gc for gc in municipios if gc]
```
Comment (israelst) [inline crawlclima/pegatweets.py]: Suggestion
```python
municipios = filter(None, gc)
```

File: crawlclima/pegatweets.py [hunk 3]
```diff
@@ -9,8 +11,17 @@
...
 
-pega_tweets.delay(week_ago.isoformat(), today.isoformat(), ['3304557', '3303302', '3106200', '4104808'], "A90")
 
+if today.isoweekday() == 5:
+    pega_tweets.delay(year_start.isoformat(), today.isoformat(), municipios, "A90")
+else:
+    pega_tweets.delay(week_ago.isoformat(), today.isoformat(), municipios, "A90")
```
Comment (israelst) [inline crawlclima/pegatweets.py]: Suggestion:
```python
pega_tweets.delay(date_from.isoformat(), today.isoformat(), municipios, "A90")
```

File: utilities/fill_locale.py [hunk 4]
```diff
@@ -13,7 +13,7 @@
 @functools.lru_cache(maxsize=None)
 def uf_geojson(uf):
     path = config('GEOJSON_PATH')
-    filename = '{}-municipalities.json'.format(uf)
+    filename = '{}-counties.json'.format(uf)
```
Comment (israelst) [inline utilities/fill_locale.py]: `br-atlas` have been updated https://github.com/carolinabigonha/br-atlas/commit/9a3839c29ed4c9e63fcffa589aa139aaca20fb6e. Now counties are called municipalities.

Comment (fccoelho) [inline utilities/fill_locale.py]: ok, I didn't know that. but for that we'd have to have the most recent
br-atlas  deployed in the server, and re-download all the files.
I think we don't have it in the server. If you want to do that, put the
repo under the /Storage directory, please.

On Fri, Oct 16, 2015 at 3:49 PM, Israel Teixeira <notifications@github.com>
wrote:

> In utilities/fill_locale.py
> <https://github.com/fccoelho/AlertaDengueCaptura/pull/11#discussion_r42276728>
> :
>
> > @@ -13,7 +13,7 @@
> >  @functools.lru_cache(maxsize=None)
> >  def uf_geojson(uf):
> >      path = config('GEOJSON_PATH')
> > -    filename = '{}-municipalities.json'.format(uf)
> > +    filename = '{}-counties.json'.format(uf)
>
> br-atlas have been updated carolinabigonha/br-atlas@9a3839c
> <https://github.com/carolinabigonha/br-atlas/commit/9a3839c29ed4c9e63fcffa589aa139aaca20fb6e>.
> Now counties are called municipalities.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fccoelho/AlertaDengueCaptura/pull/11/files#r42276728>.
>



-- 
Flávio Codeço Coelho
================
+55(21) 3799-5551
Professor
Escola de Matemática Aplicada
Fundação Getulio Vargas
Praia de Botafogo, 190 sala 312
Rio de Janeiro - RJ
22250-900
Brasil

Comment (israelst) [inline utilities/fill_locale.py]: Ok.

Comment (israelst) [inline utilities/fill_locale.py]: I forgot to mention, they are already there.

**Pull request merged**